### PR TITLE
feat(relicense): Relicense to dual-license (Unlicense or MIT)

### LINKED ISSUE #4868: [Feature] Do not use the Unlicense for scoop
## Feature Request

**Note: This is not legal advice. I am not a lawyer.**
#### Is your feature request related to a problem? Please describe.

Scoop is currently using the Unlicense and was using it since development of it began (that means Scoop is released to the public domain). I have an issue with Scoop's license - the Unlicense is **invalid** in some countries (e.g. Germany), and does not seem to be clear in a few countries (e.g. Australia). This is a problem with the Unlicense, so we should be moving to MIT or Apache because "public domain" does not exist everywhere. Also, [the Unlicense is not allowed to be used inside Google](https://opensource.google/docs/patching/#forbidden), so Google would not be allowed to make contributions to Scoop or fork it to @google. However, Google employees are important contributors to open-source, so always stay away from the Unlicense. There are also rumours going on in github/choosealicense.com#805.

This should apply to all Scoop repositories using the Unlicense (yes, that means most of the repositories).

Note that [0BSD](https://choosealicense.com/licenses/0bsd) is an exception.
#### Describe the solution you'd like

I would edit the `LICENSE` file to use a permissive license instead of the Unlicense or any other public domain dedication.
#### Describe alternatives you've considered

Using Chocolatey instead. I am also writing [CPPM](https://github.com/Tyler887/CPPM), a package manager not only for Windows, but for _any operating system that supports Python_. It's not released into the public domain but is under a permissive license.

### PR DESCRIPTION
**Pull request by niheaven**
#### Description

Just a reimplement of #4870, and some tweaks (since #4870 couldn't be edited by maintainers)

#### Motivation and Context

Closes #4868
Closes #4870 

Closes #4892

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by rasa — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+INFORMATION ABOUT THIS PROJECT'S LICENSE (LONG)
+============================================================================================
+This project ("Scoop") is free software, licensed under the Unlicense or the
+MIT license, at your option. Scoop was previously licensed under only the Unlicense,
+but was dual-licensed from version 0.2.0.
+
+Scoop comes with ABSOLUTELY NO WARRANTY. Use it at your own risk. Scoop is provided
+on an AS-IS BASIS and it's contributors disclaim all warranties.
+
+You may use, modify, distribute, sell, copy, compile, sell, or merge Scoop
```
Comment (lewis-yeung) [inline LICENSE:17]: Is the word `sell` duplicated here?

File: LICENSE [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+at your option.
+
+INFORMATION ABOUT THIS PROJECT'S LICENSE (LONG)
+============================================================================================
+This project ("Scoop") is free software, licensed under the Unlicense or the
+MIT license, at your option. Scoop was previously licensed under only the Unlicense,
+but was dual-licensed from version 0.2.0.
+
+Scoop comes with ABSOLUTELY NO WARRANTY. Use it at your own risk. Scoop is provided
+on an AS-IS BASIS and it's contributors disclaim all warranties.
```
Comment (lewis-yeung) [inline LICENSE:15]: Should `it's` be actually `its` here?

**Review by tech189 — COMMENTED** [review]
```
- **scoop-list:**: Fix date in 'Updated' column showing the months in the place of minutes ([#4880](https://github.com/ScoopInstaller/Scoop/issues/4880))
```
Extra semicolon here before 'Fix'

File: LICENSE (hunk 1 above)
Comment (niheaven) [reply LICENSE:17]: Thx, will fix it

**Review by chawyehsu — COMMENTED** [review]
LGTM

**Comment by chawyehsu** [comment]
Reminder: lack of update for the badge.

**Comment by niheaven** [comment]
> Reminder: lack of update for the badge.

It seems auto generated, and how to change it?

**Comment by bbomgardner** [comment]
It might make sense to update this in the Install repo also. Should I open a new issue?

**Pull request merged**